docs/root/configuration/best_practices/level_two.rst

```diff
@@ -5,14 +5,14 @@ Configuring Envoy as a level two proxy
 
 Envoy is a production-ready proxy, however, the default settings that are tailored for the
 edge use case may need to be adjusted when using Envoy in a multi-level deployment as a
-"level two" HTTP/2 proxy.
+"level two" proxy.
 
 .. image:: /_static/multilevel_deployment.svg
 
 **In summary, if you run level two Envoy version 1.11.1 or greater which terminates 
-HTTP/2, we strongly advise you to change the HTTP/2 configuration of your level 
+HTTP/2, we strongly advise you to change the HCM configuration of your level
```

Comment:
nit I would spell out HCM since I'm not sure users will know what this means (same elsewhere if applicable)